dyna_precice.c

```diff
@@ -2039,9 +2047,9 @@ void dyna_precice(double **cop, ITG *nk, ITG **konp, ITG **ipkonp, char **lakonp
     /* Adapter: If the coupling does not converge, read the checkpoint */
     if (Precice_IsReadCheckpointRequired()) {
       if (*nmethod == 4) {
-        Precice_ReadIterationCheckpoint(&simulationData, vold);
-        memcpy(&bj[0], &cd[0], sizeof(double) * nev);
-        memcpy(&bjp[0], &cv[0], sizeof(double) * nev);
+        Precice_ReadIterationCheckpointModal(&simulationData, bj, bjp, nev);
+        //memcpy(&bj[0], &cd[0], sizeof(double) * nev);
+        //memcpy(&bjp[0], &cv[0], sizeof(double) * nev);
```

Comment:
Please comment on why you comment-out such things, or remove them completely. We discussed previously such a cryptic commented-out line. :see_no_evil:

Reply:
Cleaning oversight again 😅